pkg/common/utils.go

```diff
@@ -492,16 +487,64 @@ func isUserMultiAttachError(err error) (codes.Code, error) {
 	return codes.Unknown, fmt.Errorf("Not a user multiattach error: %w", err)
 }
 
+// existingErrorCode returns the existing gRPC Status error code for the given error, if one exists,
+// or Unknown if one doesn't exist. Since github.com/googleapis/gax-go/v2/apierror now wraps googleapi
```

Comment:
Sorry one last nit. Can you please replace  "or Unknown if one doesn't exist." with "or an error if one doesn't exist". 

The error is the important thing. We ignore the Unknown code that is returned with the error.